strawberry/contrib/django/views.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from django.http import HttpResponseNotAllowed, JsonResponse
+from django.http.response import HttpResponseBadRequest
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+
+from asgiref.sync import async_to_sync
+from graphql import graphql
+from graphql.error import format_error as format_graphql_error
+from graphql.type.schema import GraphQLSchema
+
+
+class GraphQLView(View):
+    schema = None
+
+    def __init__(self, schema=None):
+        assert schema, "You must pass in a schema to GraphQLView"
+        assert isinstance(
```

Comment:
I am not a big fan of using `assert` for this kind of validation because they can be disabled if python is run with `-O` but it might not matter much after all? What do you think?

Reply:
I'm indifferent - will remove the assertions :)